Main.java

```diff
@@ -0,0 +1,63 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+	public static void main(String[] args) {
```

Comment:
Good tests, maybe add a test of a null argument to make it perfect.
BTW if you hadn't try working with JUnit tests yet, I think you should give it a try :)